modules/standard/aou/aou_controller.py

```diff
@@ -44,33 +48,37 @@ def inject(self, registry):
         self.items_controller = registry.get_instance("items_controller")
         self.cache_service = registry.get_instance("cache_service")
         self.command_alias_service = registry.get_instance("command_alias_service")
+        self.ts: TranslationService = registry.get_instance("translation_service")
+        self.getresp = self.ts.get_response
 
     def start(self):
         self.command_alias_service.add_alias("title", "aou 11")
         self.command_alias_service.add_alias("totw", "macro aou 171|aou 172")
         self.command_alias_service.add_alias("som", "macro aou 169|aou 383")
         self.command_alias_service.add_alias("reck", "aou 629")
         self.command_alias_service.add_alias("pets", "aou 2")
+        self.ts.register_translation("module/aou", self.load_aou_msg)
+
+    def load_aou_msg(self):
+        with open("modules/standard/aou/aou.msg", mode="r", encoding="utf-8") as f:
+            return hjson.load(f)
 
     @command(command="aou", params=[Int("guide_id")], access_level="all",
              description="Show an AO-Universe guide")
     def aou_show_cmd(self, request, guide_id):
         guide_info = self.retrieve_guide(guide_id)
 
         if not guide_info:
-            return "Could not find AO-Universe guide with id <highlight>%d<end>." % guide_id
-
-        blob = ""
-        blob += "Id: " + self.text.make_chatcmd(guide_info.id, "/start https://www.ao-universe.com/main.php?site=knowledge&id=%s" % guide_info.id) + "\n"
-        blob += "Updated: <highlight>%s<end>\n" % guide_info.updated
-        blob += "Profession: <highlight>%s<end>\n" % guide_info.profession
-        blob += "Faction: <highlight>%s<end>\n" % guide_info.faction
-        blob += "Level: <highlight>%s<end>\n" % guide_info.level
-        blob += "Author: <highlight>%s<end>\n\n" % self.format_bbcode_code(guide_info.author)
-        blob += self.format_bbcode_code(guide_info.text)
-        blob += "\n\n<highlight>Powered by<end> " + self.text.make_chatcmd("AO-Universe.com", "/start https://www.ao-universe.com")
+            return self.getresp("module/aou", "no_guide_id", {"id": guide_id})
 
-        return ChatBlob(guide_info.name, blob)
+        guide_info["id"] = self.text.make_chatcmd(guide_info.id,
+                                                  "/start https://www.ao-universe.com/main.php?site=knowledge&id=%s" % guide_info.id)
+        guide_info["aou"] = self.text.make_chatcmd("AO-Universe.com", "/start https://www.ao-universe.com")
+        guide_info["text"] = self.format_bbcode_code(guide_info.text)
+        return ChatBlob(guide_info.name, self.getresp("module/aou", "guide", {**guide_info,
+                                                                              "aou": self.text.make_chatcmd(
```

Comment:
this seems redundant given you have already set "aou" in line 76. so one of them should be removed